### DISCUSSION TITLE
Wave heights decreasing in x, even when entering shallower water

### DISCUSSION THREAD

**Discussion by lockie653** [DualSPHysics/DualSPHysics#210] [DUALSPHYSICS_V5.2]
Hi, I was wondering if anyone has experienced a reduction in significant wave height as you move along x away from the irregular wave generator in 2D SPH simulations. The attached image shows the experimental expected Hs in red and the SPH modelled Hs in blue. The bathymetry is slowly reducing the water depth in x so it would be expected to cause wave shoaling and increase wave height, however wave heights continue to decrease? I am yet to find any settings that can fix this? 

I have attached the xml file i am using along with the bathymetry file.

Any help or tips are appreciated thanks! 

![ExperimentvsModelHs](https://github.com/DualSPHysics/DualSPHysics/assets/126438029/607d7a2a-8e13-437a-8b79-82f07e847178)
[1C_Calibration_Def.xml.txt](https://github.com/user-attachments/files/16115978/1C_Calibration_Def.xml.txt)

![Bathymetry](https://github.com/DualSPHysics/DualSPHysics/assets/126438029/c9c03e68-c0c4-48dc-86f9-f2d5e425cf1d)
[Bathymetrystl.zip](https://github.com/user-attachments/files/16115980/Bathymetrystl.zip)

**Comment by iarba27** [comment]
Dear,
First, you built the piston after the bottom. This created a hole at the bottom (piston particles replaced the bottom ones). When the piston is moving, the fluid goes through the hole and the fluid particles are leaving the domain. I do expect that the water level drops.
Besides, for very long runs, I would recommend coefh=1.5, so the wave propagation is more accurate. 

May I ask why you used a peak enhancement factor equal to 0.1? It's not a common value, especially for JONSWAP spectra. 


Try the xml attached. If it does not work, then, send me also the txt with the position of the wave gauges, so I can use meausuretool. 
Regards
Corrado

[1C_Calibration_Def.xml.txt](https://github.com/user-attachments/files/16350193/1C_Calibration_Def.xml.txt)

**Reply by lockie653** [reply]
Thanks heaps Corrado. I will look into this! Still stuck on it. I will run and send you the points file within the next few days.

**Reply by lockie653** [reply]
Hi Corrado, 

The peak enhancement factor of 0.1 was a mistake, it should be the default value of 3.3.

After running your supplied file and changing the peak enhancement factor to 3.3, there was a small improvement as can be seen in the attached figure, however it is still not behaving as expected and attenuation occurs.
![CorradoHSComparison](https://github.com/user-attachments/assets/4b909059-0675-460a-8362-b96a3bdaeed5)

I have attached the points file that can be used.
[wavegauge.txt](https://github.com/user-attachments/files/16420256/wavegauge.txt)

Note that I only ran your xml file for 500s due to computational time whereas the study runs the experiment for 1680s. I have a longer one running now but expect similar results.

I also do not include the first 244s of surface elevation data in significant wave height calculations to match the experiments.

I calculate significant wave height at 4 * stdev(surf elevation data)

I have also attached this exact xml file that gave the results in the figure (green markers)

[1C_Calibration_Def.xml.txt](https://github.com/user-attachments/files/16420286/1C_Calibration_Def.xml.txt)

Thank you for looking into this,

Lochlan

**Comment by lockie653** [comment]
Update on this if anyone is interested: 
1. managed to get correct shoaling behavior by using regular wave generator.
2. Irregular wave generator still sees attenuation along the flume, may be an issue with SPH and irregular waves?
3. moved to a regular validation case so no longer need to validate the irregular case.
If anyone has validated SPH with irregular waves and experimental data id be interested to hear how it went.
Thanks,
Lochlan

**Reply by iarba27** [reply]
Dear Lochlan
sorry for my late response. I performed several runs with irregular waves without experiencing noticeable wave decay. Can you remind me the numerical setup that you are using?
Regards
Corrado
